backend/Cargo.toml

```
@@ -22,11 +22,11 @@
   tracing = { version = "0.1" }
   tracing-subscriber = { version = "0.3.18", features = [ "env-filter" ] }
 
-  strata-bridge-primitives = { git = "https://github.com/alpenlabs/strata-bridge.git" }
+  strata-bridge-primitives = { git = "https://github.com/alpenlabs/strata-bridge.git", tag = "v0.2.0-rc2" }
```

We have the `rc3` tagged today
```suggestion
  strata-bridge-primitives = { git = "https://github.com/alpenlabs/strata-bridge.git", tag = "v0.2.0-rc3" }
```